burn-import/src/onnx/to_burn.rs

```diff
@@ -152,6 +154,13 @@ impl ONNXGraph {
         for node in self.nodes {
             match node.node_type {
                 NodeType::Conv2d => graph.register(Self::conv2d_conversion::<PS>(node)),
+                NodeType::MaxPool => {
+                    if node.name == "maxpool2" {
+                        graph.register(Self::max_pool2d_conversion(node))
+                    } else {
+                        panic!("MaxPool: {} operator not supported", node.name);
+                    }
+                }
```

Comment:
We should not rely node.name. Instead we should introduce MaxPool2d node type, like it was done for Conv2d (ONNX does not have Conv2d, only Conv). There is somewhere logic that determines if it's 2d.